moml/models/mgnn/hmgnn.py

```
@@ -346,44 +521,36 @@ def forward(
-        # Ensure all graph_embeds have consistent batch dimension before cat
-        # This should be handled by the zero-node logic above ensuring num_graphs_in_batch consistency.
-        # If graph_embeds is empty (e.g. self.S = 0, though unlikely), handle torch.cat
-        if not graph_embeds:  # Should not happen if self.S > 0
-            combined_graph_pred = torch.empty(0, device=x.device if "x" in scale_data[0] else "cpu")  # Placeholder
+        if not graph_embeds:
+            combined_graph_pred = torch.empty(0, device=x.device if "x" in scale_data[0] else "cpu")
         else:
             try:
                 concatenated_graph_embeds = torch.cat(graph_embeds, dim=1)
                 combined_graph_pred = self.combined_graph_head(concatenated_graph_embeds)
             except RuntimeError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)